### ISSUE TITLE
Narrator/NVDA is not mentioning about number of results or no results found when find option is selected

### ISSUE THREAD

**Issue by LadyNaggaga** [microsoft/vscode#86819]
.NET Online REPL  is having the following accessibility issue.  Please see this [issue](https://github.com/dotnet/try/issues/109) . I have also included the details below 
> **Environment Details**
> https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
> **Browser:** Edge (41.16299.402.0)
> Chrome (67.0.3396.87)
> **Screen Reader:** Narrator
> NVDA
> 
> **Additional Details:**
> MAS Violated: MAS1.3.2
> 
> **Repro steps:**
> 
> 1. Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
> 2. Tab till “Enter focus mode” and press enter
> 3. Type any code and press F1 and tab till find and press enter
> 4. Verify Whether the Screen Reader is mentioning about number of results or no results found  when find option is selected.
> 
> **Actual Result:**
> Screen Reader is not mentioning about number of results or no results found when find option is selected.
> 
> **Expected Result:**
> Screen Reader should mention that the number of the results found or should metion that no results have found when find option is selected.
> 
> **User Impact:**
> Screen reader users find difficult in understanding if the screen reader did not mention that the number of the results or no results found when find option is selected.
> 
> **MAS Reference:**
> [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B1ee227a7-e3c5-47b8-ab18-48ca8595077b%7D)
> 
> [screen reader is not mentioning about number of results or no results found when find option is selected..pptx](https://github.com/dotnet/try/files/2188630/screen.reader.is.not.mentioning.about.number.of.results.or.no.results.found.when.find.option.is.selected.pptx)

**Comment by isidorn** [comment]
this should be fixed, closing. Also no effort was put into filling a human readable issue.
